bot/exts/help_channels/_cog.py

```diff
@@ -142,10 +145,8 @@ async def dormant_check(self, ctx: commands.Context) -> bool:
     @commands.command(name="close", aliases=["dormant", "solved"], enabled=False)
     async def close_command(self, ctx: commands.Context) -> None:
         """
-        Make the current in-use help channel dormant.
-
-        Make the channel dormant if the user passes the `dormant_check`,
-        delete the message that invoked this.
+        Make the current in-use help channel dormant and change the dynamic message
+        in the help instructions channel to match the new available channels.
```

Comment:
Closing a channel has no direct effect on which channels are available. This subtle behaviour is irrelevant to users and doesn't need to be documented here. Keep in mind this is documentation users may see (via the `!help` command) since it's a public command.

Granted, the original documentation isn't great either since it refers to an internal function. However, it's out of scope of this PR to be addressing that.